processors/sshd/misc_type.go

```diff
@@ -6,8 +6,65 @@ import (
 	"github.com/metal-toolbox/auditevent"
 
 	"github.com/metal-toolbox/audito-maldito/internal/common"
+	"github.com/metal-toolbox/audito-maldito/internal/metrics"
 )
 
+func processAcceptedPasswordEntry(config *SshdProcessorer) error {
+	matches := passwordLoginRE.FindStringSubmatch(config.logEntry)
+	if matches == nil {
+		logger.Infoln("got processAcceptedPasswordEntry log with no string sub-matches")
+		return nil
+	}
+	config.metrics.IncLogins(metrics.PasswordLogin, metrics.Success)
```

Comment:
What do you think about moving this into the lines of code you added in `processors/sshd/openssh_regex.go`?

That way, the metric always gets incremented even if we do not find any string submatches.

Reply:
That makes sense. Updated so not to include one-off metrics in `misc_type.go`